docs/cloud-messaging/receive.md

```diff
@@ -409,3 +409,97 @@ experimentalSetDeliveryMetricsExportedToBigQueryEnabled(messaging, true);
 ```
 
 Don't forget to run `yarn build` in order to export the new version of your service worker to the `web` folder.
+
+
+## (Advanced, Optional) Allowing Notification Images on iOS
+
+On Apple devices, in order for incoming FCM Notifications to display images from the FCM payload, you must add an additional notification service extension. This is not a required step.
+
+If you are using FirebaseAuth phone authentication, you must add the Firebase Auth pod to your Podfile.
+
+### Step 1 - Add a notification service extension
+
+- From Xcode top menu go to: File > New > Target...
+- A modal will present a list of possible targets, scroll down or use the filter to select "Notification Service Extension". Press Next.
+- Add a product name (use ImageNotification to follow along), set Language to Objective-C and click Finish.
```

Comment:
Is ImageNotification a literal?  What's meant by "to follow along?"

Reply:
The developer can name the Notification Service as he wants, but by naming it "ImageNotification" he will be able to copy and parse the code in the next part

Reply:
In that case I'd strongly recommend "to follow along with with this tutorial" or something like that.